docs/changelog.rst

```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+5.2.2
```

Comment:
I wonder if it should be 5.3.0 because it's a new feature?

Reply:
It should be 5.3.0 as it is a new feature.

(Aside: gidgethub was created before I moved to CalVer myself, so this isn't because I personally like SemVer. 😅)